CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+##
```

Comment:
Should we leave this as blank? We could consider adding a tentative version string with a `*` annotation to indicate that this is not yet released (Example: `1.14.5*` or `1.15.0*`). Or, we can add a placeholder release tag (Example: `UNRELEASED`). What do you think? 

cc @jhaynes

Reply:
I'd go with a placeholder for now, and we'll fill in the actual version as part of the release process. "UNRELEASED", "IN-PROGRESS", or "PENDING" would be my suggestions.

Reply:
I vote `"UNRELEASED"`